spec/features/admin_notice_spec.rb

```diff
@@ -1,30 +1,33 @@
 require 'rails_helper'
-feature 'admin tries to login' do
+feature 'admin create notice' do
   let(:user) { create(:admin) }
   let(:notice) { build(:notice) }
-  scenario 'they get alert with text "Loggade in"' do
-    visit '/logga_in'
-    fill_in 'user_username', with: user.username
-    fill_in 'user_password', with: '12345678'
-    click_button 'Logga in'
-    page.should have_css('div.alert.alert-info') # Verify we get an alert
-    find('div.alert.alert-info').text.should include('Loggade in.') # Verify that the alert states we are signed in
-    # TODO Include warden to stub login
-    #login_as(user, scope: :user)
+  let(:login) { LoginPage.new }
 
-    # Test notice
-    visit notices_path
-    find('h2#notice-headline').text.should include('Notiser')
-    find(:linkhref, new_notice_path).click
-    fill_in 'notice_title', with: notice.title
-    fill_in 'notice_description', with: notice.description
-    find(:css, "#notice_public").set(notice.public)
-    fill_in 'notice[d_publish]', with: notice.d_publish
-    fill_in 'notice[d_remove]', with: notice.d_remove
-    fill_in 'notice_sort', with: notice.sort
-    find('#submit-notice').click
+  Steps 'Create menu' do
+    And 'I sign in' do
+      login.visit_page.login(user, '12345678')
+    end
 
-    page.should have_css('div.alert.alert-info')
-    find('div.alert.alert-info').text.should include(%(#{I18n.t(:notice)} #{I18n.t(:success_create)}))
+    And 'visit menu index' do
+      visit notices_path
+      find('h2#notice-headline').text.should include('Notiser')
+      find(:linkhref, new_notice_path).click
+    end
+
+    And 'Fill out notice form' do
+      fill_in 'notice_title', with: notice.title
+      fill_in 'notice_description', with: notice.description
+      find(:css, "#notice_public").set(notice.public)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.